app/serializers/section/show_serializer.rb

```diff
@@ -6,6 +6,7 @@ def attributes(*args)
     hash[:buttons] = object.buttons if object.regular?
     hash[:description] = object.description unless object.carousel?
     hash[:cards] = object.cards if object.carousel?
+    hash[:image] = object.image if object.regular?
```

Comment:
I see no test for this? Don't we have serializer specs that would need to be updated?

Reply:
Specs updated